src/Controller/Controller_Pdf_Queue.php

```diff
@@ -278,6 +279,27 @@ protected function get_queue_tasks( $entry, $form, $notifications = [] ) {
 
 		$queue_data = apply_filters( 'gfpdf_queue_initialise', [], $entry, $form );
 
+		/**
+		 * Check if Asynchronous notification is enabled for this form.
+		 * https://docs.gravityforms.com/gform_is_asynchronous_notifications_enabled/
+		 */
+		$is_asynchronous = gf_apply_filters(
```

Comment:
Let's move this to another method in this class and then check if `! $is_asyncronous` on line 214.